rust/services/call/host/src/host/error/preflight.rs

```diff
@@ -92,11 +89,11 @@ impl From<GuestTransactError> for TransactError {
 
 #[derive(Debug, Error, PartialEq)]
 pub enum RevertError {
-    #[error("revert: <empty>")]
+    #[error("Transaction reverted: <empty>")]
```

Comment:
I wonder, would it be possible to provide some more context here, or at the very least what `<empty>` actually means in this case?

Reply:
Will do